tests/model_extend_test.py

```diff
@@ -381,11 +380,11 @@ def test_to_dict_single_zone():
     assert model_dict['rooms'][0]['properties']['energy']['program_type'] == \
         office_program.name
     assert model_dict['rooms'][0]['properties']['energy']['hvac'] == \
-        IdealAirSystem().to_dict()
+        room.properties.energy.hvac.name
 
     """
-    f_dir = 'C:/Users/chris/Documents/GitHub/energy-model-schema/app/models/samples/json'
-    dest_file = f_dir + '/model_single_zone_office.json'
+    f_dir = 'C:/Users/chris/Documents/GitHub/honeybee-schema/honeybee_schema/samples'
+    dest_file = f_dir + '/model_complete_single_zone_office.json'
     with open(dest_file, 'w') as fp:
         json.dump(model_dict, fp, indent=4)
     """
```

Comment:
I think we discussed this before. If you need code snippet like this put it in a different script and not test. If it needs to be updated on each release then it should be part of travis. Having it inside test files is not the best idea.

Reply:
I was just trying to avoid duplicating the ~30 lines of code before this that it takes to create each model. Now that things are settling down and the JSON samples generated with this code aren't likely to change as much, I'll send a PR after this one that duplicates these model-generating scripts in another file but has the un-commented snippet. This way, I can move the snippet out of the tests. I'll just have to remember that, if we make edits, I'll have to update both the tests and the sample-generating code. And, as you say, we can have Travis run the samples to make sure the sample-generating process is not broken. So should we start a `samples` folder next to the `test` folder?

Reply:
These are two different tasks:

1. Testing the samples for honeybee.
2. Generating sample schemas for honeybee-schema module.

This file is supposed to test extending the model. I think they should be fully separated. Separation of concerns is more important than code duplication.

Reply:
Great! Do you agree with making a `samples` folder next to the `tests` folder to house all of this code that generates the JSON samples for `honeybee-schema`?  Or does code like this actually belong in the `honeybee-schema` repo?

Reply:
> does code like this actually belong in the honeybee-schema repo?

That's a good question. That might actually be the right place to generate the model samples and save them. It also makes sense to have the samples updated in each release.

If you decide to keep them in this repository for now I think the folder is better to be called `scripts`.

Reply:
Ok. Let's do it under the honeybee_schema repo, then. [I added an issue](https://github.com/ladybug-tools-in2/honeybee-schema/issues/13) and I'll aim to address it soon after I've finished the other tasks on my sprint.